packages/odyssey-react/src/components/Link/Link.module.scss

```diff
@@ -11,40 +11,40 @@
  */
 
 .root {
-  color: var(--ColorTextPrimary);
+  color: var(--PrimaryTextColor);
```

Comment:
This can be reduced to `TextColor` since it's the default TextColor for Links.

(Possibly just `Color`, waiting for a weigh-in [here](https://oktawiki.atlassian.net/wiki/spaces/UX/pages/2467301041/Design+Tokens+and+How+We+Name+Them).)